### ISSUE TITLE
Open desktop Suite in larger window

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-suite#2937]
**Describe the bug**
On my PC the desktop opens quite small (like 1/3 of my screen). Could we make it bigger? And proportional to my view port maybe?

**Info:**
- Suite Version: desktop 36c7716
- OS: NixOS
- Firmware Version: 1.9.3
- Viewport: 1920 x 975

**Steps to reproduce:**
1. Launch Suite :)

**Note(s):**
Related to https://github.com/trezor/trezor-suite/issues/2938. If #2938 would be done we can close this one IMHO.

**Comment by slowbackspace** [comment]
heh so this is most likely caused by https://github.com/trezor/trezor-suite/issues/2938 (we have already implemented saving the window size, there is a bug tho), it seems that we get incorrect window dimensions (perhaps dimensions of closed window) on `before-quit` event, so instead of these wrong dimensions some default values are saved instead. I am working on a fix in #2938. Stay tuned!
Ohh and in case where there are no stored window dimensions, Raf already did some smart calculation of initial size for the window which should be large enough.

@keraf I hope you are not working on this in parallel 😁

**Comment by slowbackspace** [comment]
Hmm, I think we can actually close this since it is already implemented, but not working due to bug for which there is another issue.

cc @tsusanka

**Comment by tsusanka** [comment]
Ok by that you mean https://github.com/trezor/trezor-suite/issues/2938?

**Comment by slowbackspace** [comment]
@tsusanka it should work fine now, bug described in https://github.com/trezor/trezor-suite/issues/2938 is fixed making this feature request (for feature that is already implemented) unnecessary.
feel free to reopen PR if something's wrong 🙇
